web-client/src/global.scss

```
@@ -27,8 +27,8 @@
 
 :root {
   // Docs: https://ionicframework.com/docs/theming/themes#application-colors
-  --ion-background-color: var(--ion-color-prussian-blue);
-  --ion-background-color-rgb: var(--ion-color-prussian-blue-rgb);
+  --ion-background-color: var(--ion-color-primary);
+  --ion-background-color-rgb: var(--ion-color-primary-rgb);
   --ion-text-color: var(--ion-color-white);
   --ion-text-color-rgb: var(--ion-color-white-rgb);
 }
```

We should update Ionic's stepped colours here too:

(Ionic relies on these being sync with the background and text colours: see the [Stepped Colors docs](https://ionicframework.com/docs/theming/themes#stepped-colors))

```suggestion

  // Generated with: https://ionicframework.com/docs/theming/themes#stepped-color-generator
  // NOTE: Re-generate these whenever --ion-background-color / --ion-text-color change
  --ion-color-step-50: #0d3e59;
  --ion-color-step-100: #1a4862;
  --ion-color-step-150: #26526a;
  --ion-color-step-200: #335d73;
  --ion-color-step-250: #40677c;
  --ion-color-step-300: #4d7185;
  --ion-color-step-350: #597b8d;
  --ion-color-step-400: #668596;
  --ion-color-step-450: #738f9f;
  --ion-color-step-500: #809aa8;
  --ion-color-step-550: #8ca4b0;
  --ion-color-step-600: #99aeb9;
  --ion-color-step-650: #a6b8c2;
  --ion-color-step-700: #b3c2cb;
  --ion-color-step-750: #bfccd3;
  --ion-color-step-800: #ccd6dc;
  --ion-color-step-850: #d9e1e5;
  --ion-color-step-900: #e6ebee;
  --ion-color-step-950: #f2f5f6;
}
```